python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -1311,6 +1311,38 @@ $defs:
                 type: bool
                 default: true
                 description: Enable / Disable IPsec over HA path-group when HA is enabled.
+              ha_interfaces:
+                description: |-
+                  Local WAN HA interfaces
+                  Overwrite the default behavior which is to pick all the interfaces.
+                  Each list item supports range syntax that can be expanded into a list of interfaces.
+                  Can be used to filter uplink interfaces when there are multiple uplinks.
+                  Limitations:
+                    Either all interfaces must be uplinks or all interfaces must not be uplinks.
+                    Only one interface is supported for non uplinks.
+                type: list
+                items:
+                  type: str
+                  pattern: "Ethernet[\\d/]+"
```

Comment:
This pattern will not allow for the range syntax as mentioned in the description.

Reply:
Good catch - removed the range syntax - we can add it later